zh_cn/about/logo/index.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: page
+title: "Ruby 标识"
+lang: zh_cn
+---
+
+![Ruby 标识][logo]
+
+Ruby 标识是受版权保护的 &copy; 2006，Yukihiro Matsumoto。
+
+基于 [Creative Commons Attribution-websiteShareAlike 2.5 License][cc-by-sa] 条目协议授权。
```

Comment:
Attribution-websiteShareAlike -> Attribution-ShareAlike

("website" needs to be deleted.)